tests/E2E/test/campaigns/common_scenarios/module.js

```diff
@@ -8,7 +9,11 @@ module.exports = {
   installModule: function (client, ModulePage, AddProductPage, moduleTechName) {
     test('should go to "Module" page', () => client.goToSubtabMenuPage(Menu.Improve.Modules.modules_menu, Menu.Improve.Modules.modules_services_submenu));
     test('should set the name of the module in the search input', () => client.waitAndSetValue(ModulePage.module_selection_input, moduleTechName));
-    test('should click on "Search" button', () => client.waitForExistAndClick(ModulePage.selection_search_button));
+    test('should click on "Search" button', () => {
+      if (global.ps_mode_dev)
+        promise = client.waitForExistAndClick(AddProductPage.symfony_toolbar);
```

Comment:
can write:

```js
return client.waitForExistAndClick(AddProductPage.symfony_toolbar).then(() => client.waitForExistAndClick(ModulePage.selection_search_button));
```

Reply:
Hello @PierreRambaud ,
We can't click on the Symfony debug toolbar if the debug mode is disabled